piccolo/utils/pydantic.py

```diff
@@ -221,7 +227,15 @@ def create_pydantic_model(
         elif isinstance(column, Email):
             value_type = pydantic.EmailStr
         elif isinstance(column, Varchar):
-            value_type = pydantic.constr(max_length=column.length)
+            if not column._meta.required:
```

Comment:
Having the truthy statement first makes the logic easier to follow. For example:

```python
if something is True:
    ...
else:
    ...
```